web-app/css/openboxes.css

```diff
@@ -1,3 +1,29 @@
+:root {
+    --page-background: #F0F1F4;
+    --color-red: #C71610;
+    --color-green: #13AD60;
+    --color-yellow: #F3A157;
+
+/* BLUE*/
+    --blue-primary: #0052CC;
+    --blue-100: #F7F9FD;
+    --blue-200: #F0F6FF;
+    --blue-201: #F4F7FE;
+    --blue-300: #E9EDF4;
+    --blue-400: #7AA3F3;
```

Comment:
I think you missed some new colors from the latest PR's from react.
`--blue-301: #D0DAED;` and `--gray-401: #747C93;` were added recently

Reply:
I had an older base and also did not need those colors you mentioned, that's why I did not care about them. Gonna add them.